### DISCUSSION TITLE
Sagas

### DISCUSSION THREAD

**Discussion by gius** [Eventuous/eventuous#330] [Q&A]
Do you use sagas with Eventuous, or do you handle consecutive events flow in another way? How do you recommend to implement it?

**Comment by JulianMay** [comment]
In short: subscribe to events and invoke commands.

Take a look at https://github.com/Eventuous/eventuous/issues/176

**Reply by gius** [reply]
Thank you for the response, my point was mainly about linking proper instance-events across different aggregates - https://github.com/Eventuous/eventuous/issues/176#issuecomment-2154601809
